### ISSUE TITLE
Align the creation of Observation from Questionnaire, QuestionnaireResponse with the examples below

### ISSUE THREAD

**Issue by reeager** [qe-collaborative-services/1115-hub#202]
- Questionnaire/QuestionnaireResponse and Observation should be located in one bundle in reality. 
https://github.com/qe-collaborative-services/1115-hub/blob/f336f1858bb89f5bfdfc70445359db51b06192d7/Gravity%20Project/AHC%20HRSN%20Questionnaire.json_Update%206.25.2024.txt 
https://github.com/qe-collaborative-services/1115-hub/blob/f336f1858bb89f5bfdfc70445359db51b06192d7/Gravity%20Project/questionnaire_response_example.json

**Comment by shah** [comment]
@ratheesh-kr @raphaelvp please review and ask @reeager or me if you have questions.

**Comment by alan-francis** [comment]
@reeager , we understand we currently have the following `resourceTypes` in our bundle JSON.

- Bundle
- Consent
- Organization
- Observation
- Patient
- Encounter

And we are generating the JSON on a per patient per encounter basis. We need to add the following `resourceTypes` to this as well.

- Questionnaire
- QuestionnaireResponse

Also we believe that we need to take the data from the `screening` CSV file for generating this `resourceType`. Can you please confirm this?

@shah @raphaelvp @ratheesh-kr

**Comment by reeager** [comment]
@alan-francis No. Questionnaire and QuestionnaireResponse should not be derived from the CSV just these resourcetypes:
Bundle
Consent
Organization
Observation
Patient
Encounter

Questionnaire and QuestionnaireResponse should never be created by QCS but if Questionnaire/QuestionnaireResponse are sent to QCS via a QE, an observation resource should be derived from these resources.

**Comment by raphaelvp** [comment]
@reeager,

If Questionnaire and QuestionnaireResponse should not be derived from the CSV, please confirm if this is a requirement for FHIR Bundle submission.

**Comment by reeager** [comment]
It is not a requirement for FHIR bundle submission
